python/ribasim/ribasim/delwaq/generate.py

```diff
@@ -287,7 +339,7 @@ def _setup_boundaries(model):
 
     if model.basin.concentration.df is not None:
         for _, rows in model.basin.concentration.df.groupby(["node_id"]):
-            for boundary_type in ("Drainage", "Precipitation"):
+            for boundary_type in ("Drainage", "Precipitation", "Surface_Runoff"):
```

Comment:
Isn't this how it is defined above?

```suggestion
            for boundary_type in ("Drainage", "Precipitation", "SurfaceRunoff"):
```

Reply:
Yeah, but this parses it from our tables, which uses an underscore. We later remove the underscore in creating the boundary names.